Automattic/wp-calypso
# Help Center: add the new Help Center popover

#### Proposed Changes

* This adds a new popover to the Help Center to promote it like so: 
<img width="765" alt="image" src="https://user-images.githubusercontent.com/17054134/201738703-e63463b4-c74b-41a1-adc0-ef6157689881.png">
* It works in Calypso and the Editor, but still needs some work on Atomic.

#### Testing Instructions

1. Checkout this branch.
2. Apply this patch D92371-code.
3. Open Calypso my Home. You should see the popover.
4. Go to the Editor, you should see it too. 

If you want to reset the preference value, you can check these instructions: https://github.com/Automattic/wp-calypso/pull/62605#:~:text=you%20like%20here%3A-,To%20see%20the%20popover%20again,-%2C%20please%20untick%20%22dismissed

Fixes https://github.com/Automattic/wp-calypso/issues/69617
